pkg/idp/provider/github/github.go

```diff
@@ -250,3 +254,18 @@ func callbackURIPresent(app *githubclient.App, config provider.AppConfig) bool {
 	//TODO: check if callback URL is present
 	return true
 }
+
+func (g *Github) GetCredentialsForAuthenticatedApp(config provider.AppConfig) (string, error) {
```

Comment:
This is a stub. To not bloat this PR more the content will be added in another PR